nucypher/blockchain/eth/actors.py

```diff
@@ -294,6 +300,13 @@ def connect_condition_providers(
                     )
                 providers[int(chain_id)].add(provider)
 
+        # Ingest default RPC providers for each chain
+        for chain_id in self.domain.condition_chain_ids:
+            default_endpoints = get_healthy_default_rpc_endpoints(chain_id)
+            for uri in default_endpoints:
+                provider = self._make_condition_provider(uri)
+                providers[chain_id].add(provider)
```

Comment:
`providers` is a `set`, but do we want a bias to user-defined endpoints first, before using default endpoints ...? i.e. only if the user-defined endpoints are problematic do we then fallback to use the default ones.

Reply:
In other words, use a list.  Yes that is sensible, I'll make an update.